ui-components/src/actions/ExpandableContent.tsx

```diff
@@ -10,15 +11,16 @@ type ExpandableContentProps = {
 }
 
 const ExpandableContent = ({ children, strings, className }: ExpandableContentProps) => {
+  const classNames = ["expandable-content"]
+  if (className) classNames.push(...className.split(" "))
   const [isExpanded, setExpanded] = useState(false)
-  const rootClassNames = className ? `${className}` : undefined
 
   return (
-    <div className={rootClassNames}>
+    <div className={classNames.join(" ")}>
       {isExpanded && <div>{children}</div>}
       <button
         type="button"
-        className="button is-unstyled m-0 no-underline has-toggle"
+        className="button is-unstyled expandable-content-button no-underline has-toggle"
```

Comment:
I'm not sure if touched on this when we paired a few months ago, but these tailwind classes need to be moved to the scss file. If you take a look at other gen2 components you can get a sense for the structure of class names and variables, but I think the `MediaCard` is a good example of the naming structure (component__item as in `media-card__header`).